faucet/src/http.rs

```diff
@@ -478,7 +547,7 @@ mod tests {
         let (client, tmp_dir) = default_provider_client(AccountKeyring::Alice).await;
         set_exchange_rate(client.clone()).await;
 
-        let bob_account_id: AccountId = AccountKeyring::Bob.to_account_id();
+        let bob_account_id: AccountId = [3; 32].into();
```

Comment:
What's the reason for this change?

Reply:
The Keyring accounts are prefunded and the faucet should not send them even more funds. This change uses an unfunded account to test the faucet